game-core/src/main/java/games/strategy/triplea/ai/pro/ProPurchaseAi.java

```diff
@@ -2551,6 +2571,10 @@ private void addUnitsToPlaceTerritory(
                 isBid)) {
 
           // Place max number of units
```

Comment:
Is this comment now misplaced? It's hard to see how it relates to the next couple lines, seems like it relates to lines below it.

The comment block also seems to be blocking off a logical step, would be a good candidate for an extract method refactor.

Reply:
Yeah, removed this comment.